.github/workflows/codeql.yml

```diff
@@ -24,12 +24,12 @@ jobs:
         language: [ 'cpp' ]
 
     steps:
-    - name: Checkout repository
+    - name: Checkout repository using checkout action v4.1.7
       uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@b8efe4dc6ab6d31abe3ec159420d2a4916880800
+      uses: github/codeql-action/init@4dd16135b69a43b6c8efb853346f8437d92d3c93
```

Comment:
The latest is now 3.26.8: https://github.com/github/codeql-action/releases/tag/v3.26.8